packages/gradle/src/plugin/nodes.ts

```diff
@@ -151,15 +161,17 @@ export const makeCreateNodesForGradleConfigFile =
  */
 export const createNodes: CreateNodes<GradlePluginOptions> = [
   gradleConfigGlob,
-  async (configFile, options, context) => {
+  async (buildFile, options, context) => {
     logger.warn(
       '`createNodes` is deprecated. Update your plugin to utilize createNodesV2 instead. In Nx 20, this will change to the createNodesV2 API.'
     );
-    await populateGradleReport(context.workspaceRoot);
+    await populateGradleReport(context.workspaceRoot, [
+      getWorkspaceRootGradleBinaryPath(),
+    ]);
```

Comment:
We should be able to do something similar to v2 because all the config files are in the context

Reply:
changed